ocaml/idl/datamodel.ml

```diff
@@ -7544,6 +7544,10 @@ module VM_guest_metrics = struct
             ~ty:Bool ~default_value:(Some (VBool false)) "PV_drivers_detected"
             "At least one of the guest's devices has successfully connected to \
              the backend."
+        ; field ~qualifier:DynamicRO ~lifecycle:[]
```

Comment:
I would like to see some comment what these services are. It does not have to go into the documentation string but just providing more background what this is intended for. This could easily become something like other-config: a key-value store for all kind of unrelated information. We might want to state for example, that xapi does not use this data but just holds it for outside consumers.